### PR TITLE
Searching "@complete" in the questbook gives a list of completed quests with rewards unclaimed

### PR DESCRIPTION
**Pull request by MuXiu1997**
close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9624

### CODE REVIEW

File: src/main/java/betterquesting/api2/client/gui/panels/lists/CanvasQuestSearch.java [hunk 1]
```diff
@@ -65,7 +62,11 @@ private QuestSearchEntry createQuestSearchEntry(DBEntry<IQuestLineEntry> iQuestL
 
     @Override
     protected void queryMatches(QuestSearchEntry entry, String query, ArrayDeque<QuestSearchEntry> results) {
-        if (String.valueOf(entry.getQuest().getID()).contains(query)) {
+        if (Objects.equals(query, "@complete")) {
```
Comment (miozune) [inline src/main/java/betterquesting/api2/client/gui/panels/lists/CanvasQuestSearch.java:65]: I prefer
```java
if (query.startsWith("@") && !query.equals("@")) {
    if ("@complete".contains(query)){
```
to allow partial input.

Comment (MuXiu1997) [reply src/main/java/betterquesting/api2/client/gui/panels/lists/CanvasQuestSearch.java]: Sounds great

**Comment by miozune** [comment]
Add `!query.equals("")` so that empty query shows all quests

**Review by miozune — APPROVED** [review]
Btw now you can push branch directly to org repo instead of fork, to make testing easier.

**Comment by MuXiu1997** [comment]
> Btw now you can push branch directly to org repo instead of fork, to make testing easier.

Got it, thanks

**Pull request merged**